src/daemon/Seat.cpp

```diff
@@ -141,4 +142,11 @@ namespace SDDM {
             VirtualTerminal::jumpToVt(*nextVt, true);
         }
     }
+
+    bool Seat::canTty() {
+        OrgFreedesktopLogin1ManagerInterface manager(Logind::serviceName(), Logind::managerPath(), QDBusConnection::systemBus());
```

Comment:
This breaks for ConsoleKit2 which does not have CanTTY or similar functionality and also if neither of those are available.

As a fallback, doing `return m_name == QStringLiteral("seat0")` should do or better `return m_name.compare(QStringLiteral("seat0"), Qt::CaseInsensitive) == 0`.

Reply:
Oh, and 
`return m_name.compare(QStringLiteral("seat0"), Qt::CaseInsensitive) == 0`
I could also pair it with a `QFile::exists("/dev/tty0")`, not sure if consolekit would work without VTs, but kernels without VTs don't have /dev/tty0 so the check would be more correct if that is OK

Reply:
`manager.isValid()` + `seat.property("CanTTY").isValid()` maybe?

Reply:
I can try that, do you agree with also checking for the existence of the /dev/tty0 file in the fallback check too?

Reply:
Don't forget about FreeBSD

Reply:
Done locally